### DISCUSSION TITLE
Elhiwar Ettounsi

### DISCUSSION THREAD

**Discussion by KanekiOnMC** [iptv-org/community#813] [Show and tell]
### Channel ID

ElhiwarEttounsiTV.tn

### Stream URL

http://platin4k.eu:80/kralkerim/8TKLyJivVEA/32817

### Notes

Hello I wanna requesto to edit the link of Elhiwar Ettounsi to this one, it works 24h/24 and 7d/7

### Please confirm the following

- [x] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
EN : While we don't accept Xtream-Codes servers in our playlists, here's a free.fullspeed link using the official DM video (not 24/7) : http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/player/metadata/video/k2IuM11CZakq4ZvrUkv?embedder=https%3A%2F%2Fwww.elhiwarettounsi.com%2F

FR : Bien que l'on n'accepte pas de serveurs Xtream-Codes sur nos playlists, voici un lien free.fullspeed utilisant le direct official depuis DM (lien au dessus)

**Comment by ilyesAj** [comment]
@LaneSh4d0w the link is not working , stating that the `no streams found`
@KanekiLeChomeur since you're using an xtream server, as a normal user you will hit a [401 Unauthorized](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401) error .
is there any other suggestions for this channel ?

**Comment by BellezaEmporium** [comment]
> @LaneSh4d0w the link is not working , stating that the `no streams found`
> @KanekiLeChomeur since you're using an xtream server, as a normal user you will hit a [401 Unauthorized](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401) error .
> is there any other suggestions for this channel ?

[EN] It seems they don't often use their Dailymotion stream, though it is the one they reference in their website.

[FR] Il semblerait qu'ils utilisent très peu ce lien Dailymotion, cependant il s'agit de celui qu'ils mettent en avant sur leur site.

**Comment by freearhey** [comment]
http://platin4k.eu:80/kralkerim/8TKLyJivVEA/32817 (offline)
<img width="364" alt="image" src="https://user-images.githubusercontent.com/7253922/232234842-16ec20dd-ba37-4258-8f46-ed56114846a2.png">

http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/player/metadata/video/k2IuM11CZakq4ZvrUkv?embedder=https%3A%2F%2Fwww.elhiwarettounsi.com%2F (offline)
<img width="823" alt="image" src="https://user-images.githubusercontent.com/7253922/232234891-d60d5b73-b907-4c47-9429-0a10d96b13ac.png">
